elasticdl_client/common/args.py

```diff
@@ -177,6 +177,13 @@ def add_train_params(parser):
         help="If true, task manager supports fault tolerance, otherwise "
         "no fault tolerance.",
     )
+    add_bool_param(
+        parser=parser,
+        name="--task_timeout_tolerance",
```

Comment:
I prefer to name it `recycle_timeout_task`

Reply:
It does more than recover tasks. It also kills the corresponding worker pod.

`--worker_timeout_detection` ?

Reply:
`relaunch_timeout_worker` ?